wordle-web/src/views/WordleView.vue

```diff
@@ -1,13 +1,19 @@
 <template>
-  <v-overlay :model-value="overlay" class="align-center justify-center" persistent>
-    <v-progress-circular color="primary" indeterminate size="64" />
+  <v-overlay :model-value="overlay" class="align-center justify-center" scrim="black" persistent>
+    <WordleFlippingLetters></WordleFlippingLetters>
   </v-overlay>
 
-  <div class="text-h4 text-center">Wordle Mind Bender</div>
+  <div class="text-h4 text-center">
+    <span v-if="isWordOfTheDay"
+      >Wordle of the Day
+      <span v-if="wordOfTheDayDate"> <br />{{ wordOfTheDayDate.toLocaleDateString() }}</span>
+    </span>
+    <span v-else>Wordle Mind Bender</span>
+  </div>
 
-  <GameBoard :game="game" @letterClick="addChar" />
+  <GameBoard :game="game" />
 
-  <GameKeyboard :guessedLetters="game.guessedLetters" @letterClick="addChar" />
+  <GameKeyboard :guessedLetters="game.guessedLetters" />
```

Comment:
I think you might want to keep that letterClick event, your keyboard doesn't let you click on letters anylonger.